src/templates/Program/Session/Session.jsx

```diff
@@ -316,71 +309,64 @@ const Session = ({ data, pageContext: { pageNumber, isLastPage } }) => {
         </PageTitle>
       </Center> */}
 
-      <Container forwardedAs='div' faded>
-        <Center maxWidth='1064px' gutters='var(--container-gutter)' intrinsic>
-          <Button
-            to={pagePath}
-            tag='link'
-            outlined
-            iconFirst
-            renderIcon={<IconArrow css={backArrow} />}
-            state={{ sessionId: id }}
-            css={backButton}
-          >
-            Retour à la programmation
-          </Button>
-
-          <EventContainer>
-            <Center maxWidth='645px' gutters='var(--container-gutter)'>
-              <Stack space='40px'>
-                {!isLastPage && (
-                  <Cluster>
-                    <div>
-                      <Tag outlined>{date}</Tag>
-                      <Tag
-                        outlined
-                      >{`de ${time.beginsAt} à ${time.endsAt}`}</Tag>
-                    </div>
-                  </Cluster>
+      <Center maxWidth='1130px' gutters='var(--container-gutter)' intrinsic>
+        <Button
+          to={pagePath}
+          tag='link'
+          iconFirst
+          renderIcon={<IconArrow css={backArrow} />}
+          state={{ sessionId: id }}
+          css={backButton}
+        >
+          Retour à la programmation
+        </Button>
+
+        <EventContainer>
+          <Center maxWidth='645px' gutters='var(--container-gutter)'>
```

Comment:
680px selon la maquette
```suggestion
          <Center maxWidth='680px' gutters='var(--container-gutter)'>
```